### PR TITLE
added all files to the framework and switched import syntax

### PR DESCRIPTION
**Pull request by RoyalPineapple**

### CODE REVIEW

**Comment by brennantaylor** [comment]
Having some trouble using this as a framework. It looks like some headers got their attributes cleared. `KIFTestActor.h` is no longer part of the public headers for example.

I'm cleaning and trying again to make sure though.

**Comment by RoyalPineapple** [comment]
@brennantaylor Whoops, thanks for noting that. this was a 3 minute pass as I was leaving the office. I'll have a deeper look and clean that up tomorrow.

**Comment by RoyalPineapple** [comment]
@brennantaylor I cleaned this up a bit and was less aggressive with moving things around. I've got it building and running in my test environment. Can you try this branch again and see if it solves your issue?

**Comment by brennantaylor** [comment]
@RoyalPineapple Any update?

**Comment by RoyalPineapple** [comment]
@brennantaylor I pushed changes to the branch this afternoon and it's working in my environment. Can you give it a shot and report back?

File: Classes/KIFUIViewTestActor.m [hunk 1]
```diff
@@ -6,11 +6,12 @@
 //
 //
 
-#import "KIFUIViewTestActor.h"
-#import "UIWindow-KIFAdditions.h"
-#import "NSPredicate+KIFAdditions.h"
-#import "UIAccessibilityElement-KIFAdditions.h"
-#import "NSString+KIFAdditions.h"
+#import <KIF/KIFUIViewTestActor.h>
```
Comment (justinseanmartin) [inline Classes/KIFUIViewTestActor.m]: I think that using quotes should be the norm for inclusions within source files. If this is generating an error, I think it is something we should take a look at.

**Comment by justinseanmartin** [comment]
I rekicked the one red Travis build. The changes LGTM. Recommend that we add a test target that validates this doesn't regress in the future.

**Comment by brennantaylor** [comment]
@RoyalPineapple This lgtm now.

**Comment by phatmann** [comment]
Do we want to make a test target per @justinseanmartin's comment before we merge this?

**Comment by RoyalPineapple** [comment]
@phatmann just added one, Lets see how Travis does.

**Comment by brennantaylor** [comment]
That was quick. Thanks!

This is blocking our team from updating to xcode 7.3 so it would be nice if this were resolved soon. However I am willing to wait if that means this won't regress again.

File: scripts/ci.sh [hunk 2]
```diff
@@ -19,6 +19,7 @@ rm -rf ${PWD}/build
 # Frameworks are only supported on iOS8 and later
 if [[ ! ${SIMULATOR} =~ .*OS=7.* ]]; then
   env NSUnbufferedIO=YES xcodebuild build -derivedDataPath=${PWD}/build/KIFFramework -scheme KIFFramework -destination "platform=iOS Simulator,${SIMULATOR}" | xcpretty -c
```
Comment (justinseanmartin) [inline scripts/ci.sh]: I'd just get rid of the building command, because this is implied behavior from the test command that happens directly after this.

**Comment by phatmann** [comment]
Are we good to merge?

**Comment by RoyalPineapple** [comment]
@phatmann travis was being flakey for one configuration so i babysat it yesterday, 3 of the last 4 runs were stable while one run hung on launching the sim (I'm betting thats just general travis flakiness). Im feeling comfortable merging this now.

**Comment by phatmann** [comment]
Alright, merging now! This will solve a bunch of issues for folks, thanks for digging in and getting this done.

**Comment by phatmann** [comment]
I pushed a new release to CocoaPods with this fix.

**Pull request merged**